summertime/model/single_doc/bart_model.py

```diff
@@ -22,6 +22,8 @@ def __init__(self, device="cpu"):
     def summarize(self, corpus, queries=None):
         self.assert_summ_input_type(corpus, queries)
 
+        self.assert_summ_input_language(corpus, queries)
```

Comment:
So if `assert_summ_input_type` and `assert_summ_input_language` functions are of the same arguments, it should be better to merge them together as one function?

For example, you can put the code of checking langs in the existing `assert_summ_input_type` function and avoiding adding another line in every model?

Reply:
Addressed this in new commits!